### DISCUSSION TITLE
Unable to Submit Site to Google Search Console

### DISCUSSION THREAD

**Discussion by chipascc** [GitbookIO/community#697] [0. General bugs and questions]
In the SEO documentation page for GitBook, it says:  

> Try [submitting your site to Google](https://developers.google.com/search/docs/advanced/crawling/ask-google-to-recrawl), which essentially asks them to index it. For GitBook spaces, this will only be possible if you are using a custom domain for your space **_and if you create a TXT DNS record to confirm ownership of the domain._**

However, DNS does not allow you to have TXT record for a subdomain if you already have a CNAME record for it.  In addition, I am not able to use any of the other verification techniques that Google Search Console is asking for.  How have other people overcome this significant problem for their custom domain GitBook sites?

Thanks in advance for any suggestions / workarounds.

**Comment by byteasylum** [comment]
I'm having this problem as well. I just [posted an issue](https://github.com/orgs/GitbookIO/discussions/797) about SEO issues with Gitbook. 

https://docs.gitbook.com/published-documentation/seo

I really need a solution for this. I definitely don't want to go back to using Wordpress. Gitbook is a lot nicer.

**Comment by nnachtegael** [comment]
Same issue for me
I definitely don't want to go back to using Wordpress

**Comment by gregberge** [comment]
> However, DNS does not allow you to have TXT record for a subdomain if you already have a CNAME record for it. In addition, I am not able to use any of the other verification techniques that Google Search Console is asking for. How have other people overcome this significant problem for their custom domain GitBook sites?

Hi @chipascc, I don't know what is your provider but it's possible to add as many TXT records independently from having a CNAME record. So it should not be a issue.
